test/integration/scheduler_perf/config/performance-config.yaml

```diff
@@ -1048,6 +1071,7 @@
       maxClaimsPerNode: 10
   - name: 2000pods_100nodes
     labels: [performance, fast]
+    threshold: 7
```

Comment:
@pohly, please follow up on improving this and the remaining of the DRA tests.

Reply:
This is classic DRA. Performance of that is less relevant. Perhaps no threshold should be recorded for it at all?

What I will focus on for 1.32 is the structured parameters variant. It needs to become more realistic for device attributes and requests, then various optimization ideas can be tried out (tracked in https://github.com/kubernetes/kubernetes/issues/113701).

Reply:
ah ok, since it's somewhat planned for removal, let's not add thresholds for it. And maybe remove all of the tests cases?

Reply:
We could stop running them, but perhaps not remove quite yet?

Having them around was useful once to see that disabling queuing hints really had a negative impact, but probably now they aren't worth the on-going compute cycles.

Reply:
I removed thresholds for DRA-related tests. I think we can stop running them in a separate PR by removing the labels from workloads.

Reply:
Ack.